tests/pos-custom-args/captures/cc-poly-varargs.scala

```diff
@@ -12,8 +12,4 @@ def either[T1, T2, Cap^](
     src2: Source[T2, Cap]^{Cap^}): Source[Either[T1, T2], Cap]^{Cap^} =
   val left = src1.transformValuesWith(Left(_))
   val right = src2.transformValuesWith(Right(_))
-  race[Either[T1, T2], Cap](left, right)
```

Comment:
The extension methods can be tested as well. I remember they have similar issues.

Reply:
It causes an infinite regress now, looking into it.